app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,32 @@
+
+
+
+
+
+
+<div class="footer-top">
+  <div class="footer-links-text" >
+    <a href="#"><i > Invest </i></a>
+    <a href="#"><i > Develop </i></a>
+    <a href="#"><i > About Us </i></a>
+    <a href="#"><i > Contact Us </i></a>
```

Comment:
the navbar right now has 'Invest, Raise, Portfolio and contact' - do we want this stuff in both the navbar and the footer? (i'm ok if so, just wanna simplify things if possible and hear rationale)